package.json

```
@@ -30,13 +30,14 @@
     "husky": "^5.0.9",
     "prettier": "^2.2.1",
     "semantic-release": "^17.3.9",
-    "stylelint": "^14.2.0",
-    "typescript": "^4.1.5"
+    "stylelint": "^14.3.0",
+    "typescript": "^4.5.5"
   },
   "dependencies": {
-    "stylelint-config-prettier": "^8.0.1",
-    "stylelint-config-standard": "^20.0.0",
-    "stylelint-scss": "^3.17.2"
+    "stylelint-config-prettier": "^9.0.3",
+    "stylelint-config-standard": "^24.0.0",
+    "stylelint-config-standard-scss": "^3.0.0",
+    "stylelint-scss": "^4.1.0"
```

```suggestion
    "stylelint-config-standard-scss": "^3.0.0"
```

Presuming that the other dependencies aren't required now we've added this one